src/__tests__/Avatar.tsx

```diff
@@ -5,11 +5,10 @@ import {px, render, behavesAsComponent, checkExports} from '../utils/testing'
 import {render as HTMLRender, cleanup} from '@testing-library/react'
 import {axe, toHaveNoViolations} from 'jest-axe'
 import 'babel-polyfill'
-import {COMMON} from '../constants'
 expect.extend(toHaveNoViolations)
 
 describe('Avatar', () => {
-  behavesAsComponent({Component: Avatar, systemPropArray: [COMMON]})
+  behavesAsComponent({Component: Avatar, systemPropArray: []})
```

Comment:
Do we need to pass an array to `systemPropArray` if it's empty?
```suggestion
  behavesAsComponent({Component: Avatar})
```

Reply:
had to slightly tweak the typing of behavesAsComponent, but with an easy fix this was possible 👍